Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -191,6 +207,33 @@ - (void)ensureContentConstraints
 {
     [NSLayoutConstraint deactivateConstraints:self.contentContraints];
 
+    if (self.attachmentToApprove) {
+        self.contentView.hidden = YES;
+        self.attachmentApprovalView.hidden = NO;
+
+        self.contentContraints = @[
+            [self.attachmentApprovalView autoSetDimension:ALDimensionHeight toSize:300.f],
```

Comment:
just checking: 300 is good on iPhone5 and iPhone7+?

Reply:
Good q - it's part of the design discussion I'm hoping to have in iOS standup once our retreat is over.